tests/test_cuda_bd_exchange_mover.py

```diff
@@ -461,7 +470,7 @@ def hif2a_rbfe_state() -> InitialState:
 
 @pytest.mark.parametrize(
     "steps_per_move,moves",
-    [(1, 500), (5000, 5000)],
```

Comment:
Find it sketchy that this changed so much. Note that for float32 it only takes ~950 moves to move once, while for float64 it takes 5900~ moves. But the average acceptance in water is 1/7000 moves, so maybe this is to be expected?? Need to think some more about

Reply:
Over how many total samples? Do you get comparable acceptance rates in biased deletion only?

Reply:
There are 6000 samples (by samples I assume you mean an attempted moved) and this is biased deletion in the context of ligand + complex. Currently only know the acceptance rate of biased deletion, still running the targeted simulations.

Reply:
This test started failing after https://github.com/proteneer/timemachine/pull/1205/commits/476b6a7ef9d7361cab52985e96b23ecd42c1c653 and after investigating its simply due to the change in the ordering of using the rng (previously was quaternion then translation, now translation then quaternion). This test seems very flaky (and slow), however I do think it is important to have this test logic as otherwise we don't have a test case of what is our most common case (RBFE). And now that bias deletion and targeted insertion share code, it is important to validate both of these cases.

An additional thing we could do is triple the number of moves and move it to nightly.

Reply:
IIRC the average acceptance rate was something like 1/6000 to begin (sometimes it can happen faster, sometimes slower, but on average its 1/6000), so this doesn't seem too out of place.